manifests/s/Splashtop/SplashtopPersonal/3.5.0.3/Splashtop.SplashtopPersonal.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Splashtop.SplashtopPersonal

```

```suggestion
PackageIdentifier: Splashtop.Splashtop.Personal
```
Because `Personal` is an edition, it should be made a sub-package